src/Util/CapabilitiesUtil/CapabilitiesUtil.spec.js

```diff
@@ -0,0 +1,153 @@
+/*eslint-env jest*/
+import OlLayerTile from 'ol/layer/tile';
+import { CapabilitiesUtil } from '../../index';
+
+const layerTitle =  'OpenStreetMap WMS - by terrestris';
+const capabilitiesObj = {
+  version: '1.1.0',
```

Comment:
Do we also support 1.3.0?

Reply:
Yes we do since the OL capabilities parser does. I changed version this in this object used for testing `getLayersFromWmsCapabilties`.